website/static/js/nodeControl.js

```diff
@@ -196,9 +200,9 @@ var ProjectViewModel = function(data, options) {
         bootbox.confirm({
             title: 'Create identifiers',
             message: '<p class="overflow">' +
-                'Are you sure you want to create a DOI and ARK for this ' +
-                $osf.htmlEscape(self.nodeType) + '? DOI and ARK identifiers' +
-                ' are persistent and will always resolve to this page.',
+                'Are you sure you want to create a DOI for this ' +
+                $osf.htmlEscape(self.nodeType) + '? DOI  identifier' +
```

Comment:
Change to "A DOI" -- DOI stands for "Digital Object Identifier" so the extra identifier is redundant, but adding an "A" will make it a complete sentence